Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Tables"
 uuid = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 authors = ["quinnj <quinn.jacobd@gmail.com>"]
-version = "1.2.2"
+version = "1.2.3"
```

Comment:
I think it should be 1.3 unless @quinnj thinks that the fact that methods you add were missing was a bug.

Reply:
I think it's fair to say it's a bug, since `Tables.Columns` seemed to promise a table interface but was broken due to this issue.  I don't have a strong opinion though, if you want 1.3 let me know and I'll change it.

Reply:
I would leave it for @quinnj to decide 😄.

Reply:
I'd be inclined to use a more restrictive definition of "bugfix". Patch releases should only include changes that really fix errors or invalid (misleading/dangerous) results. Changes in behaviors (such as the one here) should definitely be avoided as much as possible as they may break code. People should be able to update to patch releases without fearing breakage.

Reply:
I've increased the version number.